config/default.py

```diff
@@ -25,6 +25,11 @@
 SECURITY_MSG_PASSWORD_NOT_PROVIDED = ("Invalid username/password combination", "error")
 SECURITY_MSG_USER_DOES_NOT_EXIST = ("Invalid username/password combination", "error")
 
-logging.basicConfig(level=logging.DEBUG)
+logging.basicConfig(level=logging.WARNING)
```

Comment:
What do these logger settings do differently than before? Why WARNING instead of DEBUG?